client/my-sites/upgrades/components/domain-warnings/index.jsx

```diff
@@ -519,6 +520,40 @@ export default React.createClass( {
 				section="domain-management" />;
 	},
 
+	pendingTransfer() {
+		const domains = this.getDomains().filter( domain => domain.pendingTransfer );
+
+		if ( domains.length !== 1 ) {
+			return null;
+		}
+
+		const domain = domains[ 0 ];
+
+		const compactNotice = this.translate( '{{strong}}%(domain)s{{/strong}} is pending transfer.', {
+				components: { strong: <strong /> },
+				args: { domain: domain.name }
+			} ),
+			fullNotice = this.translate(
```

Comment:
Don't think a comment is needed. This rule is not in the default whitelist, so it has to be passed through props.